content/docs/dvclive/api-reference/log.md

```diff
@@ -46,8 +46,9 @@ environment variables (when used alongside `DVC`).
 
 - `val` - The value to be added in the `name` column of a new row.
 
-- `step` (`None` by default) - The value to be added in the `step` column of a
-  new row. If `None`, the value of the internal `_step` property will be used.
+- `step` (`None` by default) - The value to be added in the
+  [`step`](/doc/dvclive/api-reference/get_step) column of a new row. If `None`,
+  the value of the internal `step` property will be used.
```

Comment:
```suggestion
- `step` (`None` by default) - The value to be added in the `step` column of a
  new row. If `None`, the value of
  [`dvclive.get_step()`](/doc/dvclive/api-reference/get_step) will be used.
```

What do you think of this? I'd prefer to use the public methods than referencing internal properties if possible.